objects/src/accounts/delta/mod.rs

```diff
@@ -190,12 +190,14 @@ fn validate_nonce(
                 ))
             },
         }
-    } else if nonce.is_some() {
-        return Err(AccountDeltaError::InconsistentNonceUpdate(
-            "nonce updated for empty delta".to_string(),
-        ));
```

Comment:
This is the temporary bug fix.

Reply:
I think this check specifically has been discussed in the context of some of pioneer's projects and a possibility was to remove it altogether (#765)

Reply:
even better :)